app/services/xccdf_tailoring_file.rb

```diff
@@ -58,9 +58,16 @@ def rule_group_selections_builder(xml)
     end
   end
 
-  def selections_builder(xml)
+  def value_builder(xml)
+    @set_values.each do |value_ref_id, value|
+      xml[XCCDF].send('set-value', value, idref: value_ref_id)
```

Comment:
Calling `send` is a big red flag, especially with a fixed method name. Please look up the documentation of the class you're interacting with and find an alternative solution.

Reply:
Besides `send`, the only other option I've seen is to use `tag!` like:
`xml[XCCDF].tag! 'set-value', value, idref: value_ref_id`

But I don't think this is supported anymore because it doesn't work correctly. I can look into this more after we finish importing values into the DB but just wanted to leave a comment here.

Reply:
@skateman What do you think about this? ^ 
Should I keep `send` and explain it with a comment or still try to find another option?

Reply:
I guess it should be fine

Reply:
Okay, thanks I'll still add a comment explaining why we're using it